packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/CodeBlock.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState } from "react";
```

Comment:
do you think we could use only one CodeBlock component or are they too different ?

Reply:
The both have slightly different styles props.
and this CodeBlock has icons and a copy function :)